src/openforms/authentication/contrib/yivi_oidc/tests/oidc_plugins/test_plugins.py

```diff
@@ -44,14 +44,15 @@ def test_yivi_process_claims_with_dots_in_path(self):
         plugin = oidc_register[oidc_client.identifier]
         AttributeGroupFactory.create(
             name="know_attributes",
+            uuid="00000000-0000-0000-0000-12345678abcd",
```

Comment:
Use https://www.uuidgenerator.net/ or a plain python shell to generate actually valid uuid4's

These examples aren't.